app/models/profile_declaration.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+class ProfileDeclaration < ApplicationRecord
+  delegated_type :declarable, types: %w[EarlyCareerTeacherProfileDeclaration MentorProfileDeclaration]
+  belongs_to :participant_declaration
+  belongs_to :lead_provider
+
+  scope :ect_profiles, lambda {
```

Comment:
This is a bit messy, but that might be partly because of the formatting. Can you do it in Arel?

Reply:
Scope has been pulled out of this PR.